src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -3382,6 +3353,40 @@ nofile soft/hard 65536
 
 In addition, we recommend to have at least 16GB RAM for 15.000 nodes.
 
+=== Housekeeping
+
+The Scheduler provides a housekeeping mechanism that periodically removes finished jobs
+from the Scheduler Portal. It can also remove jobs and all its data from the database to save space.
```

Comment:
from the **Scheduler and Workflow Execution portals**. It **also removes** jobs and all its data from the database to save space.

Reply:
@nebilbenmabrouk I updated accordingly, can you approve the PR please ?